templates/settings/payment-manage.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Dokan Settings Payment Template
+ *
+ * @package dokan
+ */
+?>
+
+<form method="post" id="payment-form"  action="" class="dokan-form-horizontal">
+
+    <?php wp_nonce_field( 'dokan_payment_settings_nonce' ); ?>
+
+    <?php
+    $method = dokan_withdraw_get_method( $method_key );
```

Comment:
Again don't process things inside template. Do all these works where you are including this template. Also , the template part of `'global/dokan-error'` that you have imported below should be imported from the function.